CI: flaky test failure of nil pointer of prometheus vec

```
=== RUN   TestStoreAsync
2022/07/28 09:57:26.664273 juicefs[19487] <INFO>: Disk cache (/tmp/diskCache/): capacity (1 MB), free ratio (10%), max pending pages (2) [disk_cache.go:93]
2022/07/28 09:57:26.664572 juicefs[19487] <INFO>: Found 1 staging blocks (0 bytes) in /tmp/diskCache/ with 58.686µs [disk_cache.go:547]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x16899bd]
goroutine 107 [running]:
github.com/prometheus/client_golang/prometheus.(*CounterVec).WithLabelValues(0x3eef078ac16d25d1, {0xc00052a3f0, 0x1e78665, 0x22a1ee0})
	/home/travis/gopath/pkg/mod/github.com/prometheus/client_golang@v1.11.0/prometheus/counter.go:246 +0x1d
github.com/juicedata/juicefs/pkg/chunk.(*cachedStore).put.func1()
	/home/travis/gopath/src/github.com/juicedata/juicefs/pkg/chunk/cached_store.go:351 +0x451
github.com/juicedata/juicefs/pkg/utils.WithTimeout.func1()
	/home/travis/gopath/src/github.com/juicedata/juicefs/pkg/utils/utils.go:70 +0x2f
created by github.com/juicedata/juicefs/pkg/utils.WithTimeout
	/home/travis/gopath/src/github.com/juicedata/juicefs/pkg/utils/utils.go:69 +0xe5
FAIL	github.com/juicedata/juicefs/pkg/chunk	2.595s
```
https://app.travis-ci.com/github/juicedata/juicefs/builds/253736389

--- a/pkg/chunk/cached_store.go
+++ b/pkg/chunk/cached_store.go
@@ -671,6 +671,7 @@ func NewCachedStore(storage object.ObjectStorage, config Config, reg prometheus.
 	if config.DownloadLimit > 0 {
 		store.downLimit = ratelimit.NewBucketWithRate(float64(config.DownloadLimit)*0.85, config.DownloadLimit)
 	}
+	store.initMetrics()
 	store.bcache = newCacheManager(&config, reg, func(key, fpath string, force bool) bool {
 		if force {
 			return store.addDelayedStaging(key, fpath, time.Time{}, true)
@@ -711,7 +712,11 @@ func NewCachedStore(storage object.ObjectStorage, config Config, reg prometheus.
 			}
 		}()
 	}
+	store.regMetrics(reg)
+	return store
+}
 
+func (store *cachedStore) initMetrics() {
 	store.cacheHits = prometheus.NewCounter(prometheus.CounterOpts{
 		Name: "blockcache_hits",
 		Help: "read from cached block",
@@ -746,12 +751,9 @@ func NewCachedStore(storage object.ObjectStorage, config Config, reg prometheus.
 		Name: "object_request_data_bytes",
 		Help: "Object requests size in bytes.",
 	}, []string{"method"})
-	store.initMetrics(reg)
-
-	return store
 }
 
-func (store *cachedStore) initMetrics(reg prometheus.Registerer) {
+func (store *cachedStore) regMetrics(reg prometheus.Registerer) {
 	if reg == nil {
 		return
 	}

